cfgov/ask_cfpb/scripts/replace_unsupported_characters.py

```diff
@@ -0,0 +1,31 @@
+from ask_cfpb.models.pages import AnswerPage
+
+
+unsupported = [u'\x92', u'\x93', u'\x94', u'\x96']
+
+
+def fix(text):
+    text = text.replace(u'\x92', "'")
+    text = text.replace(u'\x93', '"')
+    text = text.replace(u'\x94', '"')
+    text = text.replace(u'\x96', '-')
+    return text
+
+
+def run():
+    for page in AnswerPage.objects.all():
+        fixed = False
+
+        for string in unsupported:
+            if string in page.question:
+                page.question = fix(page.question)
+                fixed = True
+            if string in page.answer:
+                page.answer = fix(page.answer)
+                fixed = True
+            if string in page.snippet:
+                page.snippet = fix(page.snippet)
+                fixed = True
+
+        if fixed:
+            page.save_revision().publish()
```

Comment:
Can we be sure that any page we find with bad characters can be published? It might be the case currently, but perhaps not in the future.

Reply:
Right -- so unless they're planning on making any changes in the next week or so, I think this is safe as a one-time thing.  That's why I'm not sure it should be merged into the codebase or not; I was hoping to run it just once so that the updates to the Ask export can get merged.